autoware.repos

```diff
@@ -47,3 +47,7 @@ repositories:
     type: git
     url: https://github.com/CPFL/car_demo.git
     version: e364448fad421cb6244c9f828f978d8d877dcbf9
+  vehicles/sd:
+    type: git
+    url: https://github.com/streetdrone-home/SD-VehicleInterface.git
+    version: master
```

Comment:
@EfimiaPanagiotaki-StreetDrone could you tag this? It'll be easier to track bugs if the users can tell us what version of the interface they are using. Thanks!

Reply:
This is for the master of Autoware. Do we want to allow the master of Autoware to also use the master of its dependencies, or do we want to always depend on an explicit version?